client/Android/Studio/build.gradle

```diff
@@ -1,21 +1,25 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
-Properties properties = new Properties()
+Properties releaseProperties = new Properties()
 File file = new File('release.properties')
 if (file.canRead()) {
-    properties.load(new FileInputStream(file))
+    releaseProperties.load(new FileInputStream(file))
 }
    
 if (!hasProperty('RELEASE_STORE_FILE')) {
-    ext.RELEASE_STORE_FILE='nokeyfile'
+    ext.RELEASE_STORE_FILE=releaseProperties['RELEASE_STORE_FILE']
 }
 if (!hasProperty('RELEASE_KEY_ALIAS')) {
-    ext.RELEASE_KEY_ALIAS=''
+    ext.RELEASE_KEY_ALIAS=releaseProperties['RELEASE_KEY_ALIAS']
```

Comment:
missing default value if the `releaseProperties` file also does not contain that. use `properties.get('RELEASE_STORE_FILE', 'nokeyfile')` or similar